src/acquire.c

```diff
@@ -98,6 +98,8 @@ acquire_map_read(const struct AcquireRuntime* self_,
     self = containerof(self_, struct runtime, handle);
     struct vfslice_mut slice = make_vfslice_mut(channel_read_map(
       &self->video[istream].sink.in, &self->video[istream].monitor.reader));
+    EXPECT(self->video[istream].monitor.reader.status == Channel_Ok,
+           "Expected an unmapped reader. See acquire_unmap_read().");
```

Comment:
I'm assuming unmapped reader (i.e., `Channel_Expected_Unmapped_Reader`) is the most common failure mode? The other option for status from `channel_read_map` is `Channel_Error` in the case of overflow. The error message here is misleading in that case.

Reply:
fwiw, `Channel_Error` indicates something is wrong with the code in `channel.c` itself; it never happens due to a user error. That said, I'll improve the error checking here.